certif/app/routes/authenticated/sessions/add-student.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+
+export default class AuthenticatedSessionsDetailsAddStudentRoute extends Route {
+  async model(params) {
+    const session = await this.store.findRecord('session', params.session_id);
+    const certificationCenter = this.modelFor('authenticated');
+    const students = await this.store.findAll('student',
+      { adapterOptions : { certificationCenterId: certificationCenter.id } },
+    );
+    return { sessionId: session.id, students };
```

Comment:
Est-ce que ça ça fonctionnerait ?
```suggestion
    const certificationCenter = this.modelFor('authenticated');
    const students = await this.store.findAll('student',
      { adapterOptions : { certificationCenterId: certificationCenter.id } },
    );
    return { sessionId: params.session_id, students };
```

Reply:
Effectivement, ce findRecord semble inutile